templates/ethstore.html

```diff
@@ -0,0 +1,524 @@
+                </tr>
+                <tr>
+                  <td>Sum effective balances (EB):</td>
+                  <td>Sum of all effective balances at epoch SE of all validators in VS</td>
+                </tr>
+                <tr>
+                  <td>ETH.STORE:</td>
+                  <td>(RS/EB)*365</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+
+          <br />
+          <p>
+            Calculations for <strong><span aria-ethereum-date="{{ .Data.YesterdayTs }}" aria-ethereum-date-format="dd MMMM yyyy"></span> 12:00:23 UTC - <span aria-ethereum-date="{{ addI64 .Data.YesterdayTs 86400 }}" aria-ethereum-date-format="dd MMMM yyyy"></span> 12:00:22 UTC:</strong>
+          </p>
+
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>Variable</th>
+                  <th>Description</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Start epoch (SE):</td>
+                  <td>Epoch {{ .Data.StartEpoch }}</td>
+                </tr>
+                <tr>
+                  <td>End epoch (EE):</td>
+                  <td>Epoch {{ addUI64 .Data.StartEpoch 225 }}</td>
+                </tr>
+                <tr>
+                  <td>Active validators (VS):</td>
+                  <td>in timeframe [{{ .Data.StartEpoch }},{{ addUI64 .Data.StartEpoch 225 }})</td>
+                </tr>
+                <tr>
+                  <td>Sum rewards (RS):</td>
+                  <td>{{ .Data.YesterdayRewards }} Ether</td>
+                </tr>
+                <tr>
+                  <td>Sum effective balances (EB):</td>
+                  <td>{{ .Data.YesterdayEffectiveBalance }} Ether</td>
+                </tr>
+                <tr>
+                  <td>ETH.STORE:</td>
+                  <td>{{ .Data.ProjectedAPR }} % = ({{ .Data.YesterdayRewards }} / ({{ .Data.YesterdayEffectiveBalance }})) * 365</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+
+        <button type="button" class="collapsible">Is the ETH.STORE code Open-Source?</button>
+        <div class="content">
+          <p>Yes, you can run the calculations yourself with our open source ETH.STORE tool and verify the integrity of the ETH.STORE listed here. Check out <a href="https://github.com/gobitfly/eth.store/" target="_blank">ETH.STORE on Github</a>.</p>
+        </div>
+      </section>
+
+      <div style="padding-bottom: 5vh;"></div>
+    </div>
+  </div>
+{{ end }}
+
+{{ define "js" }}
+  <script src="/js/highcharts/highstock.min.js"></script>
+  <script src="/js/highcharts/highcharts-global-options.js"></script>
+  <script>
+	var coll = document.getElementsByClassName("collapsible");
+	var i;
+
+	for (i = 0; i < coll.length; i++) {
```

Comment:
Why declare i above?

Reply:
The entire JS code was copied from another project and barely touched, won't change for now